tests/test_xiaomi.py

```diff
@@ -1664,6 +1639,56 @@ async def test_xiaomi_e1_roller_commands_2(zigpy_device_from_quirk, command, val
     )
 
 
+@pytest.mark.parametrize(
+    "quirk",
+    (zhaquirks.xiaomi.aqara.switch_acn047.AqaraT2Relay,),
+)
+async def test_aqara_t2_relay_endpoint_1(zigpy_device_from_quirk, quirk):
```

Comment:
Instead of giving the quirk via parametrize, we should be able to give `1` and `2` for the endpoint and avoid the duplicated test.
If you need help with this, I can push the changes later or so.